setup.py

```diff
@@ -52,7 +52,7 @@
     "pytest==5.3.2",  # MIT license
     "pytest-cov==2.8.1",  # MIT
     "pytest-watch==4.2.0",  # MIT
-    "xain-sdk==0.5.0",  # Apache License 2.0
+    "xain-sdk @ git+https://github.com/xainag/xain-sdk.git@development",  # Apache License 2.0
```

Comment:
Does it work without the `#egg=xain-sdk`?

Reply:
it seems so, i just wanted to test that, installed everything freshly and built a new docker image and it worked.

Reply:
Alright, I will approve it